tests/functional/end-to-end.bats

```diff
@@ -381,53 +386,62 @@ test_report_format_impl() {
 	fi
 
 	run valgrind --leak-check=full --show-leak-kinds=all --errors-for-leak-kinds=all --error-exitcode=1 ${SELINT_PATH} -c configs/default.conf -r -s policies/check_triggers
+	echo ${output}
 	[ "$status" -eq 0 ]
 }
 
 @test "nesting_gen_req" {
 	run ${SELINT_PATH} -c configs/default.conf -e W-002 -E -s policies/misc/nesting.*
+	echo ${output}
 	[ "$status" -eq 0 ]
 	count=$(echo ${output} | grep -o "W-002" | wc -l)
 	echo "Status: $status, Count: $count (expected 1)"
-	echo $output
+	echo ${output}
 	[ "$count" -eq 1 ]
 	count=$(echo ${output} | grep -o "foo_data_t" | wc -l)
 	echo "Status: $status, Count: $count (expected 1)"
-	echo $output
+	echo ${output}
 	[ "$count" -eq 1 ]
 	count=$(echo ${output} | grep -o "foo_log_t" | wc -l)
 	echo "Status: $status, Count: $count (expected 0)"
-	echo $output
+	echo ${output}
 	[ "$count" -eq 0 ]
 }
 
 @test "disable comment" {
 	run ${SELINT_PATH} -c configs/default.conf -F -e W-002 -E -s policies/misc/disable.*
+	echo ${output}
 	[ "$status" -eq 0 ]
 	count=$(echo ${output} | grep -o "W-002" | wc -l)
 	echo "Status: $status, Count: $count (expected 0)"
-	echo $output
+	echo ${output}
```

Comment:
As above, this one now seems redundant to the one above.

Reply:
(In this case, the subsequent ones in the function all follow a run, so for this test, it's just this line that I think needs dropping).